docs/source/development/coding_conventions.rst

```diff
@@ -292,6 +292,89 @@ If this is a link it should refer to where the link goes.
 	<element id="mds.nav.schema-editor"/>
 	<element id="task.sidebar.tasks-toggle-active"/>
 
+
+Common Conventions for API creation
+###################################
+
+HTTP methods and endpoints
+--------------------------
+
+We use following common pattern for endpoints:
```

Comment:
this is not needed, when developer defines request mapping in rest controller, he/she provides only the end of url